mealy/error_analyzer.py

```diff
@@ -474,13 +487,13 @@ def inverse_transform_thresholds(self):
         return self._error_clf_thresholds
 
     # TODO: rewrite this method using the ranking arrays
-    def error_node_summary(self, leaf_selector='all_errors', add_path_to_leaves=False, print_summary=False):
+    def error_node_summary(self, leaf_selector='all', add_path_to_leaves=False, print_summary=False):
         """ Return summary information regarding input nodes.
 
         Args:
             leaf_selector (int or list or str): the desired leaf nodes to visualize. When int it represents the
-                number of the leaf node, when a list it represents a list of leaf nodes. When a string, the valid values
-                are either 'all_error' to plot all leaves of class 'Wrong prediction' or 'all' to plot all leaf nodes.
+                number of the leaf node, when a list it represents a list of leaf nodes. When a string, the valid value
+                is 'all' to plot all leaf nodes.
```

Comment:
Should be made clearer, like what we do in `_get_leaf_selector`:
```suggestion
   leaf_selector: None, int or array-like
   The leaves whose information will be returned
       * int: Only return information of the leaf with the corresponding id
       * array-like: Only return information of the leaves corresponding to these ids
       * None (default): Return i nformation of all the leaves
```